vector/src/main/res/values-fr/strings.xml

```diff
@@ -1012,6 +1012,8 @@
         <item quantity="one">%d j</item>
         <item quantity="other">%d j</item>
     </plurals>
+    <string name="room_encrypted">Salon crypté</string>
+    <string name="room_status_online">En ligne</string>
```

Comment:
These strings are used as `contentDescription`, I want to talk with you about them